libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/SessionState.java

```diff
@@ -71,15 +71,15 @@ int secondsSinceLastReroute() {
     if (lastRerouteDate() == null) {
       return -1;
     }
-    long diffInMs = lastRerouteDate().getTime() - new Date().getTime();
+    long diffInMs = new Date().getTime() - lastRerouteDate().getTime();
```

Comment:
`SessionState.secondsSinceLastReroute()` is called during `Telemetry.pushEvent()` which happens ~20s after the original reroute event. Since we're using `new Date()` here, that means `secondsSinceLastReroute` is computed as the seconds from the send event to the `lastRerouteDate`, rather than the seconds from the original reroute time to the last reroute date.

See also my comment above (https://github.com/mapbox/mapbox-navigation-android/pull/348/files#r145172948) about storing the last reroute date.